src/Components/Tile/Tile.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import './Tile.css';
+
+class Tile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    }
+    this.canvas = React.createRef();
+  }
+
+  drawTile( canvas ) {
+    let ctx = canvas.getContext('2d');
+    let {dx, dy, image, x, y} = this.props;
+    ctx.drawImage(image, x, y, dx, dy, 0, 0, 300, 150);
+  }
+
+  componentDidMount() {
+    this.drawTile(this.canvas.current)
+  }
+
+  render() {
+    return <canvas
+      ref={this.canvas}
+      style={{
+        width: "100px",
+        height: "100px",
+        borderColor: "black",
+        borderStyle: "double"
+      }}>
```

Comment:
Awesome, this definitely will be easier to work with instead of doing the single canvas route like I had mocked up. 

After pulling down and interacting with the dev tools to see the component layout I think this format will certainly be able to house all of the functionality we need